src/pybind/mgr/dashboard/controllers/iscsi.py

```diff
@@ -489,11 +486,7 @@ def _target_portal_deletion_required(old_portal_host, old_portal_ip_list, new_po
 
     @staticmethod
     def _target_deletion_required(target, new_target_iqn, new_target_controls):
-        if target['target_iqn'] != new_target_iqn:
-            return True
-        if target['target_controls'] != new_target_controls:
-            return True
-        return False
+        return target['target_iqn'] != new_target_iqn
```

Comment:
We should be backward compatible here by looking into the `api_version` returned by the ceph-iscsi `/api/settings` endpoint.

Reply:
Done